### PR TITLE
Map certs with VOMS attr before local daemons (SOFTWARE-3489)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: config/condor_mapfile.osg [hunk 1]
```diff
@@ -1,3 +1,4 @@
+GSI ".*,/[A-Za-z0-9]*/Role=[A-Za-z0-9]*/Capability=NULL.*" GSS_ASSIST_GRIDMAP
```
Comment (bbockelm) [inline config/condor_mapfile.osg]: I'm not sure about the `.*` at the end -- can we have `$` instead?

Comment (bbockelm) [reply config/condor_mapfile.osg]: Ah, also, this might not work with nested groups (`/cms/uscms`).

Comment (brianhlin) [reply config/condor_mapfile.osg]: I think that would possibly break FIFE jobs. IIRC, they have a ton of VOMS FQANs

Comment (brianhlin) [reply config/condor_mapfile.osg]: Good point about the nested groups. How deep can they go?

Comment (bbockelm) [reply config/condor_mapfile.osg]: Pretty deep.  I would just allow `/` in the character specification.

Comment (brianhlin) [reply config/condor_mapfile.osg]: I've fixed the match expression to support nested groups and limited the matches for subsequent VOMS FQANs. Are there any other characters that are supported in VO names, group names, roles, and capabilities that we need to add?

File: config/condor_mapfile.osg [hunk 2]
```diff
@@ -1,3 +1,4 @@
+GSI ".*,/[/A-Za-z0-9\.]*/Role=[A-Za-z0-9]*/Capability=NULL[,/A-Za-z0-9\.=]*$" GSS_ASSIST_GRIDMAP
```
Comment (bbockelm) [inline config/condor_mapfile.osg]:
```suggestion
GSI ".*,/[/A-Za-z0-9\.]*/Role=[A-Za-z0-9\.]*/Capability=NULL$" GSS_ASSIST_GRIDMAP
```

Two small tweaks:
1. I think `.` is valid for Roles.
2. Nothing is permitted past the capability.

Comment (brianhlin) [reply config/condor_mapfile.osg]: > Nothing is permitted past the capability.

Except additional VOMS FQANs, which come in comma separated

Comment (brianhlin) [reply config/condor_mapfile.osg]: Oh duh, it just matches the last VOMS FQAN and any preceding ones are caught in the `.*`.

**Pull request merged**